### PR TITLE
Update Subscribe, About, Terms & Contact urls

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2023-08-22 at 9 23 35 AM
Screen Shot 2023-08-22 at 9 23 59 AM
Screen Shot 2023-08-23 at 7 22 33 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Add Subscribe Link to Navigation](https://trello.com/c/rjg9fnPq/631-add-subscribe-link-to-navigation)

**Comment by jwade1327** [comment]
[Update Contact Page and About Page Content](https://trello.com/c/Mj85CsZw/630-update-contact-page-and-about-page-content)

File: sites/cpgnext.com/config/navigation.js [hunk 1]
```diff
@@ -24,14 +23,14 @@ const resources = [
...
-  { href: '/page/pw-about-us', label: 'About' },
-  { href: '/page/pw-contact-us', label: 'Contact' },
+  { href: '/page/cpgnext-about-us', label: 'About' },
+  { href: '/page/cpgnext-contact-us', label: 'Contact' },
 ];
 
 const mobileMenu = {
   primary: [...topics],
   secondary: [
-    subscribe,
```
Comment (Shinsina) [inline sites/cpgnext.com/config/navigation.js:34]: Is there are reason we opted to remove this outright? This looks like exactly what would have been included here had `./subscribe` been updated to include the appropriate link

**Review by Shinsina — CHANGES_REQUESTED** [review]

File: sites/cpgnext.com/config/navigation.js (hunk 1 above)
Comment (jwade1327) [reply sites/cpgnext.com/config/navigation.js:34]: @Shinsina it is replaces with the url below 👇  line 33. They are using 2 different tracking code in their dragon forms to determine which link is clicked (the footer is different)

Comment (Shinsina) [reply sites/cpgnext.com/config/navigation.js:34]: @jwade1327 Can we remove the `subscribe` file then? (Assuming it goes unused elsewhere), seems silly to leave a file in there entirely unused.

**Review by Shinsina — CHANGES_REQUESTED** [review]

**Review by Shinsina — APPROVED** [review]

**Pull request merged**